scriptmodules/emulators/mupen64plus-testing.sh

```diff
@@ -1,23 +1,29 @@
 rp_module_id="mupen64plus-testing"
 rp_module_desc="N64 emulator MUPEN64Plus (Testing)"
-rp_module_menus="4+"
+rp_module_menus="2+"
 rp_module_flags="!odroid"
 
 function depends_mupen64plus-testing() {
-    getDepends libsamplerate0-dev libspeexdsp-dev libsdl2-dev
+    if ! hasPackage libsdl2-dev && isPlatform "rpi"; then
+        rp_callModule sdl2 install_bin
+    fi
```

Comment:
this code is not needed - it reverted my earlier change to just use libsdl2-dev with getDepends.